data/brands/amenity/vending_machine.json

```diff
@@ -101,7 +103,8 @@
       "tags": {
         "amenity": "vending_machine",
         "brand": "Bild",
-        "name": "Bild"
+        "name": "Bild",
+        "vending": "parcel_pickup"
```

Comment:
Are you sure about this one?  A quick google search looks like this might be a newspaper vending macine.